src/components/Router/index.js

```diff
@@ -66,7 +66,16 @@ class CrossDeviceMobileRouter extends Component {
   setConfig = (actions) => (data) => {
     const {token, steps, documentType, step, woopraCookie} = data
     setWoopraCookie(woopraCookie)
-    if (!token || jwtExpired(token)) return this.setError()
+    if (!token) {
```

Comment:
This conditional feels overly defensive. If we want to enforce a token it should be on the desktop rather than on the cross device client

Reply:
yeah I would drop it